src/Components/Blazor/DevServer/src/Server/Startup.cs

```diff
@@ -33,7 +33,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.MimeTypes = ResponseCompressionDefaults.MimeTypes.Concat(new[]
                 {
                     MediaTypeNames.Application.Octet,
-                    WasmMediaTypeNames.Application.Wasm
+                    "application/wasm",
```

Comment:
A string literal? You can *do* that??

![](https://media1.giphy.com/media/26ufdipQqU2lhNA4g/giphy.webp?cid=790b76115d2ee15c6358675a73ff0270&rid=giphy.webp)

Reply:
That's not enough indirection, please make a static property backed by a constant.

Reply:
You've just been visited by the #LiteralGang - upvote 20 times or else you have to take everything literally for a week.